hw3/01-characters.js

```diff
@@ -1,4 +1,24 @@
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
+const section = document.querySelector('section');
 
-fetch(url);
+fetch(url)
+  .then((response) => response.json())
+  .then((data) => {
+    for (const character of data) {
+      const card = document.createElement('card');
+      const image = document.createElement('img');
+      image.src = character.imageUrl;
```

Comment:
I forgot to do this too, but a run through Wave shows 53 errors. They are all for alt text. A good reminder for us all to add alt text. Though... I wonder what an efficient way to do this would be? Perhaps using the full name of the character for the alt text. Or maybe "Headshot of + fullname" maybe that would sound too rememtitive on a screen reader? Food for thought